coroutines-codelab/build.gradle

```diff
@@ -92,6 +103,8 @@ buildscript {
                 "androidx.arch.core:core-testing:$lifecycle_version",
                 "androidx.work:work-testing:$work_version",
 
+                // Coroutines testing
+                "org.jetbrains.kotlinx:kotlinx-coroutines-test:$coroutines_android_version",
```

Comment:
is the comma needed?

Reply:
Prefer trailing commas in multi line list literals, that way the next item only causes one item diff (and it compiles right away).